packages/usa-nav/src/test/template.html

```diff
@@ -24,13 +24,13 @@
           </button>
           <ul id="basic-nav-section-one" class="usa-nav__submenu">
             <li class="usa-nav__submenu-item">
-              <a href="#">Navigation link</a>
+              <a href="javascript:void(0)">Navigation link</a>
```

Comment:
Changing to `javascript:void(0)` allowed me to simulate links to pages without having the test leave the current page.